resources/build/run-required-test-builds.sh

```diff
@@ -58,6 +58,10 @@ function triggerTestBuilds() {
         local job=${test_build%_Jenkins}
         echo "  -- Triggering build configuration $job/$branch on Jenkins"
         triggerJenkinsBuild "$job" "$branch" "$force"
+      elif [ "${test_build:(-7)}" == "_GitHub" ]; then
+        local job=${test_build%_GitHub}
+        echo "  -- Triggering GitHub action build $job/$branch"
+        triggerGitHubActionsBuild true "$job" "$branch"
```

Comment:
In due course, we are going to want to track these on status.keyman.com, probably.